lsp-methods.el

```diff
@@ -855,7 +855,8 @@ directory."
    (when (featurep 'projectile) (projectile-project-root))
```

Comment:
If I open `/usr/include/stdio.h` directly and expect `projectile-project-root` to return nil (not to start a language server), what can I do to prevent it from returning `default-directory`